### PR TITLE
added some better questions :)

### LINKED ISSUE #5: Add questions to questions files in questions_dir
_This will be a running issue to let people know how to submit questions_

Want to contribute code-related questions for the bot to ask? This is your chance. Go for it.

---

A few rules:
- Questions must be code related
- The goal is to create conversation and substance. Avoid yes/no answers.
- Add questions to the questions file that makes the most sense.
  - a back end question would go in `questions_backend.py`
  - a pop culture question would go in `questions_pop_culture.py`
- Must fit in Twitter's 280 character limit
---

All questions added should look like the below
```python
'question goes here',  # short answer goes here, add a link if it's not a short answer
```

### Proposed workflow:
1. Fork this repo
2. Add question to applicable `question` file keeping to the existing style/conventions:
   * Each question should be in its own line
   * Surrounded by single quotes (`'`)
   * Keep the trailing comma
3. Once you are done and checked for typos, commit it & push your work to your fork
4. Submit a PR against this repo

Someone will look it over and, if deemed appropriate, will merge your PR.

Thanks for contributing!

---

**Here's some things to help spark ideas**
wordcloud

---
*Don't forget to follow the bot on [Twitter](https://www.twitter.com/CodeQBot) to see the hard work you put in.*

### PR DESCRIPTION
**Pull request by ghost**
added some new question to questions_acronyms.py, questions_back_end.py, questions_front_end.py, questions_pop_culture.py, questions_version_control.py

### CODE REVIEW

File: questions_front_end.py [hunk 1]
```diff
@@ -37,6 +37,9 @@
     'How do you make a link open in a new tab?',  # target="_blank"
     'Why should you add rel="noreferrer" to a target="_blank"? \n\ninspired by a tweet from @addyosmani',  # https://twitter.com/addyosmani/status/1234055782896979968
     'What is lazy loading? What\'s it used for?',  # https://twitter.com/addyosmani/status/1227619409625174016?s=20
+    'What is the difference between SASS and SCSS?',
```
Comment (VickiLanger) [inline questions_front_end.py]: This is a great question!

File: questions_pop_culture.py [hunk 2]
```diff
@@ -15,6 +15,9 @@
     'In "Silicon Valley" Season 3 Episode 1, what language is being used?\n@SiliconHBO',  # https://www.reddit.com/r/SiliconValleyHBO/comments/4gbra7/i_got_the_silicon_valley_s03e01_code_to_compile/)
     'What day is celebrated as Programmer\'s day? Why?\n#ProgrammersDay',  #
     '🐠In Mr. Robot, what is the name of Elliot\'s betta fish? @whoismrrobot #MrRobot',  # Qwerty
+    'Who was the first Disney Princess? @DisneyStudios' #Disney
+    'Who created Mickey Mouse? @DisneyStudios #Disney' #Disney #MickeyMouse
+    'Why is Hacktoberfest done only once in a year? @Hacktoberfest #Hacktoberfest2020'
```
Comment (VickiLanger) [inline questions_pop_culture.py]: This is a great question. Remember to follow the example in issue #5 
```python
'question goes here',  # short answer goes here, add a link if it's not a short answer
```
And the below to keep in line with Python syntax
   * Each question should be in its own line
   * Surrounded by single quotes (`'`)
   * Keep the trailing comma

File: questions_pop_culture.py [hunk 3]
```diff
@@ -15,6 +15,9 @@
     'In "Silicon Valley" Season 3 Episode 1, what language is being used?\n@SiliconHBO',  # https://www.reddit.com/r/SiliconValleyHBO/comments/4gbra7/i_got_the_silicon_valley_s03e01_code_to_compile/)
     'What day is celebrated as Programmer\'s day? Why?\n#ProgrammersDay',  #
     '🐠In Mr. Robot, what is the name of Elliot\'s betta fish? @whoismrrobot #MrRobot',  # Qwerty
+    'Who was the first Disney Princess? @DisneyStudios' #Disney
```
Comment (VickiLanger) [inline questions_pop_culture.py]: This is a fun question, but is it code-related? 
Code Questions bot is meant to help developers learn code things together.

File: questions_pop_culture.py [hunk 4]
```diff
@@ -15,6 +15,9 @@
...
+    'Who created Mickey Mouse? @DisneyStudios #Disney' #Disney #MickeyMouse
```
Comment (VickiLanger) [inline questions_pop_culture.py]: This is a fun question, but is it code-related?

**Review by VickiLanger — CHANGES_REQUESTED** [review]
Lots of great questions! Thanks for adding them. If you wouldn't mind, please address the comments I left.

Also, I'd like to add you to the Contributors list. Do. you have a Twitter handle you'd like me to include?

**Comment by ghost** [comment]
Thank you

**Review by VickiLanger — APPROVED** [review]
All adjustments made. Ready to merga

**Pull request merged**